release-notes/kvm/v11.1.0.md

```diff
@@ -3,3 +3,8 @@
 ### net-exporter [v1.6.0](https://github.com/giantswarm/net-exporter/blob/master/CHANGELOG.md#160-2020-01-29)
 
 - Allow disabling TCP DNS check.
+
+### nginx-ingress-controller [0.27.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v120-2020-01-13)
```

Comment:
This link and structure seems incorrect.
```suggestion
### nginx-ingress-controller v0.28.0([Giant Swarm App v1.3.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v130-2020-01-30))
```

Reply:
Does that apply also to the net-exporter above?

Reply:
`net-exporter` does not have an app version - it just has a single version because it is our component. Not a "packaged" app.

And yes it's confusing and not intuitive :D

Reply:
I see 👍

Reply:
I think 0.27.1 was/is OK but have to admit I don't know details/plans for 11.1.0 platform release

nginx 0.28.0 is really fresh/new as of today, next legacy cluser-operator (0.22.1-dev) changes to include nginx IC 0.28.0 are still WIP https://github.com/giantswarm/cluster-operator/pull/909

Reply:
0.27.1 related changelog in clustert-operator I think was wrongly removed from cluster-operator, and replaced with include net-exporter - probably @jgsqware assumed nginx stuff was leftover changelog from previous release while it was preparation for next cluster-operator release.

Reply:
@kopiczko v11.1.0 is active and includes cluster-operator 0.22.0 https://github.com/giantswarm/releases/blob/master/kvm.yaml#L18-L34
which includes nginx IC app 1.2.0 with nginx-ingress-cluster 0.27.1

nginx IC app 1.3.0 was only created today, it includes nginx-ingress-cluster 0.28.0, and PR to have it included in current WIP cluster-operator 0.22.1-dev is https://github.com/giantswarm/cluster-operator/pull/909 which might make it's way to 11.1.1 platform release for kvm

So, IMO the above change @MarcelMue suggested for these 11.1.0 release notes should canceled or reverted if already included in this PR.